baselines/nesterov/jax/submission.py

```diff
@@ -43,19 +43,19 @@ def init_optimizer_state(workload: spec.Workload,
 def create_lr_schedule_fn(
```

Comment:
seems like it's the exact same code in nesterov jax and pytorch submission files, maybe extract it into a util to avoid any LR scheduling bugs

Reply:
keeping separate for easier forking